frame/staking-rewards/src/lib.rs

```diff
@@ -59,16 +59,21 @@ pub mod pallet {
 	};
 	use composable_traits::{
 		currency::{BalanceLike, CurrencyFactory},
-		staking::RewardPoolConfiguration::RewardRateBasedIncentive,
+		staking::{RewardPoolConfiguration::RewardRateBasedIncentive, DEFAULT_MAX_REWARDS},
 		time::DurationSeconds,
 	};
 	use frame_support::{
-		pallet_prelude::*, traits::UnixTime, transactional, BoundedBTreeMap, PalletId,
+		pallet_prelude::*,
+		traits::{fungibles::Transfer, UnixTime},
+		transactional, BoundedBTreeMap, PalletId,
 	};
 	use frame_system::pallet_prelude::*;
 	use sp_arithmetic::{traits::One, Permill};
-	use sp_runtime::{traits::BlockNumberProvider, PerThing};
-	use sp_std::{collections::btree_map::BTreeMap, fmt::Debug, vec::Vec};
+	use sp_runtime::{
+		traits::{AccountIdConversion, BlockNumberProvider},
+		PerThing, Perbill,
```

Comment:
nit: If i didn't overlook smth., `PerThing` isn't used in this file and could be removed.